deploy/.env.dev

```diff
@@ -0,0 +1,3 @@
+NEXT_PUBLIC_RPC_URL=https://ghostnet.tezos.marigold.dev/
```

Comment:
IMO the name of the folder is a bit weird as it's not really related to deploy; but config files. So maybe `config` would be better I think